development-tools/clinic.rst

```diff
@@ -2087,6 +2087,8 @@ The generated glue code looks like this:
 
    Getters and setters must be declared as separate functions.
    The *value* parameter for a "setter" is added implicitly by Argument Clinic.
+   The docstring is also supported for :c:type:`PyGetSetDef` functions like
+   other methods, but it must be declared at ``@getter``.
```

Comment:
Maybe this wording is better?
```suggestion
   It is possible to create a docstring for the property by adding it to
   the ``@getter``.
```

I also noticed that there doesn't seem to be any section specifically for docstrings, even though there are some mentions about them in the tutorial and in the other howtos.  It might be useful to link to another part of the docs that explains how to add docstrings, but in order to do that we either need to add a label or create a new howto that explains how to do it and link to that.